guide/14-deep-learning/how_retinanet_works.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<blockquote>Figure 2. Different kinds of pyramid networks [1]</blockquote><p>These are not different kinds of networks but instead these are architectures. </p><p>Suggestion:</p><p>Various architectures for object detection</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Here, architecture and network can be use interchangeably.</p><p> </p><p>These are various pyramid network/architectures -&gt; can be used for object detection and also segmentation</p><p>Focus is on the pyramid structure more than being used for object detection.</p>